src/Panel/Ui/Drawer.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Kirby\Panel\Ui;
+
+/**
+ * @package   Kirby Panel
+ * @author    Nico Hoffmann <nico@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ * @since     6.0.0
+ * @unstable
+ */
+class Drawer extends Component
+{
+	public function __construct(
+		string $component = 'k-drawer',
+		string|null $class = null,
+		public string|null $icon = null,
+		public array|null $options = null,
+		string|null $style = null,
```

Comment:
`Question` Why did you add the class and style args here again? They would already be included in the $args list and passed on properly otherwise, right?